docs/resources/rediscloud_subscription_peering.md

```diff
@@ -39,7 +42,19 @@ resource "rediscloud_subscription_peering" "example" {
    subscription_id = rediscloud_subscription.example.id
    provider = "GCP"
    gcp_project_id = "cloud-api-123456"
-   network_name = "cloud-api-vpc-peering-example"
+   gcp_network_name = "cloud-api-vpc-peering-example"
+}
+
+output "gcloud_peering_cmd" {
+  value = <<-EOF
+  gcloud compute networks peerings create \
+  ${rediscloud_subscription_peering.example.redis_network_name} \
```

Comment:
Maybe this one now needs to be `gcp_redis_network_name`

Reply:
Thanks for spotting this.  I made the change in my HCL working example and thought I had transferred.